README.md

```diff
@@ -328,7 +328,8 @@ A third party security audit was performed by Cure53 in 4Q2018; the [full report
 
 ### Reporting security issues
 
-__If you are reporting a security issue, please reach out discreetly at security@containerd.io__.
+- __Report a new vulnerability on [Github](https://github.com/containerd/containerd/security/advisories/new)__
+- __Reach out discreetly at security@containerd.io__
 
```

Comment:
Can we just link https://github.com/containerd/project/blob/main/SECURITY.md (and update the file there)

Reply:
I'll draft this for now and re-open when the change to containerd/project gets in. I'll open it in an hour or so

Reply:
https://github.com/containerd/project/pull/101

Reply:
@AkihiroSuda This PR now references the new flow at containerd/project, thanks!